components/KiboSearchSuggestion.vue

```diff
@@ -0,0 +1,217 @@
+<template>
+  <div v-show="visible" class="custom-search-header">
+    <div class="search-container">
+      <div class="search-product left">
+        <SfLoader v-if="loading" :class="{ loader: loading }" :loading="loading">
+          <div>{{ $t("Searching......") }}</div>
+        </SfLoader>
+        <div class="suggestion-title"><h3>Top Suggestions</h3></div>
+        <div class="result-list">
+          <div v-if="result.products && result.products.length > 0" class="product-card-list">
+            <template v-for="(product, index) in result.products">
+              <KiboProductCard
+                :key="index"
+                class="product_card_details"
+                :image="product.productImageUrls[0]"
+                :title="product.productName"
+                image-width="97px"
+                image-height="97px"
+                :link="localePath(`/product/${product.productCode}`)"
+                :wishlist-icon="false"
+                @closeSearchBox="closeSearchProductClick()"
+              />
+            </template>
+          </div>
+          <div v-else class="no-record">
+            <div v-if="!loading">No Result found......</div>
+          </div>
+        </div>
+      </div>
+      <div class="vertical-line"></div>
+      <div class="search-category right">
+        <SfLoader v-if="loading" :class="{ loader: loading }" :loading="loading">
+          <div>{{ $t("Searching......") }}</div>
```

Comment:
Avoid hardcoding text here

Reply:
as per VSF it's ok. no need to add extra variable.